installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -6,13 +6,27 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Use the following procedures to install an {product-title} cluster using the Agent-based Installer.
```

Comment:
I just noticed that the page in the navigation on the left uses lowercase. I'll change this now to be capitalized. Are there other instances you saw that use lowercase? Let me know and I can fix them.

Reply:
I do not recall seeing instances that used lowercase. I was responding to the fact that the exist published documentation uses lowercase, while your updates were using upper case. [1] [2] I think this might be a combination of lowercase appearing in content, while also appearing in the topic map.

[1]https://docs.openshift.com/container-platform/4.12/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html
[2] https://docs.openshift.com/container-platform/4.12/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html